src/main/java/com/thevoxelbox/voxelsniper/brush/type/redstone/SetRedstoneFlipBrush.java

```diff
@@ -45,6 +47,15 @@ public void handleCommand(String[] parameters, Snipe snipe) {
         }
     }
 
+    @Override
+    public List<String> handleCompletions(String[] parameters, Snipe snipe) {
+        if (parameters.length == 1) {
+            String parameter = parameters[0];
+            return super.sortCompletions(Stream.of("n", "s", "ns", "e", "world", "ew"), parameter, 0);
```

Comment:
Why would `world` be treated as east/west? I'd remove it, that is no cardinal direction.